src/index.ts

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable import/prefer-default-export */
+
+export { default as BXButton } from './components/dropdown/dropdown';
```

Comment:
Did you mean `'./components/button/button'`?

Reply:
Oops good catch @cal-smith!